frontend/src/components/ToDoItem/ToDoItem.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react';
+import { ItemList } from '../../types/common';
+import './styles.css';
+
+const ToDoItem = (props: {
```

Comment:
I like this name, maybe we can expand it to List => ToDoList too?

Reply:
fixed